posts/how-to-integrate-sourcegraph-into-your-workflows.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: How to integrate Sourcegraph into your workflows
 author: marek-zaluski
-tags: [video, sourcegraph, ide, git]
+tags: [video, sourcegraph, IDE, git]
```

Comment:
I notice that the tags make tag entries title case and changing to `IDE` vs `ide` does nothing, but for a future state if we change it would be nicer if this were `IDE` on the tag page (maybe worth doing only if we have a critical mass of items tagged with "IDE").

Reply:
Yeah, we can pass whatever string logic we need!